lib/taurus/qt/qtgui/panel/taurusvalue.py

```diff
@@ -91,7 +91,7 @@ def setModel(self, model):
         try:
             config = self.taurusValueBuddy().getLabelConfig()
         except Exception:
-            config = 'label'
+            config = '{attr.label}'
```

Comment:
see https://github.com/taurus-org/taurus/pull/646/files#r156952262

Reply:
I think, it is allowed for labelConfig